### ISSUE TITLE
Missing class org.jetbrains.java.decompiler.code.ModifierList

### ISSUE THREAD

**Issue by omotorin** [Earthcomputer/kotlin-decompiler#2]
I cloned the project, but it was not compiled. ClassWriter.java and ClassesProcessor.java are import the org.jetbrains.java.decompiler.code.ModifierList class that is missing.
Could you please add this one to the repo?

**Comment by dshadowwolf** [comment]
Agreed. Looks like, possibly, an accidental deletion - I found it in the [git history](https://github.com/Earthcomputer/kotlin-decompiler/blob/2d356ce29d71ce7ed9a00e7987eac4f8b384d963/src/org/jetbrains/java/decompiler/code/ModifierList.java).

**Comment by ahmedyarub** [comment]
So the files was not added at the end?

**Comment by Earthcomputer** [comment]
Unfortunately I don't have these files locally anymore. This project was an experiment which was very incomplete. [Upstream](https://github.com/Vineflower/vineflower/) has actually added a Kotlin decompiler plugin which is more developed and I would recommend using that instead.
